content/news/playtest-20150524.md

```diff
@@ -0,0 +1,55 @@
+It has been a [month of busy development](https://github.com/OpenRA/OpenRA/pulse/monthly). Time to get back to testing.
```

Comment:
I don't think so. However I thought it shows this project isn't dead even though when the next playtest gets postponed because of unforseen consequences so it isn't necessarily a bit thing.